git-branchless-init/src/lib.rs

```diff
@@ -606,8 +606,7 @@ fn command_init(
     main_branch_name: Option<&str>,
 ) -> EyreExitOr<()> {
     let mut in_ = BufReader::new(stdin());
-    let repo = Repo::from_current_dir()?;
-    let mut repo = repo.open_worktree_parent_repo()?.unwrap_or(repo);
+    let mut repo = Repo::from_current_dir()?;
 
     let default_config = Config::open_default()?;
     let readonly_config = repo.get_readonly_config()?;
```

Comment:
This PR changes `readonly_config` to be read from the current worktree instead of the root. (It's used for modifying the root config to include `branchless/config`) This doesn't seem to break any tests, and also `uninstall` uses the current worktree instead of the root. I'm not sure if this is a problem or not, since I'm not clear on how much of the config file init/uninstall process is covered in the worktree tests.